e3sm_diags/plot/enso_diags_plot.py

```diff
@@ -48,15 +48,6 @@
 ENSO_BORDER_PADDING_MAP = (-0.07, -0.025, 0.17, 0.022)
 
 
-def _save_plot_scatter(fig: plt.Figure, parameter: EnsoDiagsParameter):
-    """Save the scatter plot using the shared _save_single_subplot function."""
-    _save_main_plot(parameter)
-
-    # Save the single subplot using shared helper (panel_config=None for full figure)
-    if parameter.output_format_subplot:
-        _save_single_subplot(fig, parameter, 0, None, None)
```

Comment:
Moved these kinds of functions at the bottom because they are the last called private function, makes it easier to read file.